api/v1alpha/api.proto

```diff
@@ -133,24 +133,31 @@ enum PodState {
 }
 
 // Pod describes a pod's information.
+// If a pod is in Embryo, Preparing, AbortedPrepare state,
+// only id and state will be returned.
+//
+// If a pod is in other states, the pod manifest and
+// apps will be returned when 'detailed' is true in the request.
+//
+// Valid pid and networks are only returned when a pod is in Running.
 message Pod {
-        // ID of the pod, in the form of a UUID, required.
+        // ID of the pod, in the form of a UUID.
         string id = 1;
 
-        // PID of the pod, optional, only valid if it's returned by InspectPod(). A negative value means the pod has exited.
+        // PID of the pod.
```

Comment:
Evidently I missed this earlier, but while you're here: we should probably describe which pid exactly this is, since a pod is multiple processes.